ARCHITECTURE.md

```diff
@@ -117,12 +117,22 @@ This table keeps track of the amendments enabled on each network.
 | Key                  | Definition                                               |
 |----------------------|----------------------------------------------------------|
 | `amendment_id`       |The amendment id.                                         |
-| `networks`           |The network(s) where the amendment has been enabled.         |
+| `networks`           |The network(s) where the amendment has been enabled.      |
 | `ledger_index`       |The ledger where the amendment has been enabled.          |
 | `tx_hash`            |The transaction hash where the amendment has been enabled.|
 | `date`               |The date and time when the amendment has been enabled.    |
 
 
+### `amendments_incoming`
```

Comment:
Can we merge `amendments_enabled` and `amendments_incoming`? They seem to contain the same unique index and have similar data.

Reply:
Should we rename it or keep it as `amendments_enabled`?

Reply:
I was thinking the same thing but just struggle to figure out a name

Reply:
Could be `amendment_networks` or `amendment_status` or something